client/me/security-2fa-enable/index.jsx

```diff
@@ -329,7 +328,6 @@ class Security2faEnable extends React.Component {
 				{ this.renderInputHelp() }
 
 				<FormVerificationCodeInput
-					autoFocus
```

Comment:
Is there a particular reason you removed this?

Reply:
It was throwing a linting error because it causes accessibility issues.